.travis.yml

```diff
@@ -1,28 +1,29 @@
 language: bash
-os: linux
-sudo: enabled
+dist: focal
 env:
   global:
     - LC_ALL: C.UTF-8
     - LANG: C.UTF-8
 
-install:
+addons:
+  snaps:
+    - name: snapcraft
+      channel: stable
+      classic: true
+    - name: http
+    - name: transfer
+    - name: lxd
+      channel: stable
+
+script:
   - sudo apt update
-  - sudo apt install -y snapd
-  - sudo snap install lxd --channel 3.0/stable
-  - sudo snap install snapcraft --candidate --classic
   - sudo /snap/bin/lxd waitready
-  - sudo /snap/bin/lxd.migrate -yes
   - sudo /snap/bin/lxd init --auto
   - mkdir -p "$TRAVIS_BUILD_DIR/snaps-cache"
-script:
-  - export PATH=/snap/bin:$PATH
-  - sudo snapcraft cleanbuild
-  - sudo cp *.snap "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
+  - sudo snapcraft --use-lxd
 after_success:
-  - sudo snap install transfer
-  - timeout 180 sudo /snap/bin/transfer "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
+  - cp *.snap "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
+  - timeout 180 transfer "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
```

Comment:
btw, we could get rid of this line as well. The underlying service will go offline on 30th October, so may just get rid of it now.

Reply:
I will do it in the next PR.